frontend/src/App.tsx

```diff
@@ -99,6 +100,11 @@ const App = (): React.ReactElement => {
                   component={Default}
                   requiredRoles={[UserRole.Admin, UserRole.Subscriber]}
                 />
+                <Route
+                  exact
+                  path={Routes.PREVIEW_REVIEW_TEST}
+                  component={PreviewReviewTest}
+                />
```

Comment:
TODO: Create ticket to remove this route and other references to this page after integration